lib/ClangImporter/ClangModuleDependencyScanner.cpp

```diff
@@ -265,17 +265,34 @@ ModuleDependencyVector ClangImporter::bridgeClangModuleDependencies(
       swiftArgs.push_back(IncludeTree);
     }
 
+    // Ensure we get the "real", vfs-remapped path to the modulemap
+    // becuase it is used as an explicit input for the resulting compilation
+    // task.
+    auto externalModuleMapPathRef =
+        getClangInstance().getFileManager().getFileRef(
+            clangModuleDep.ClangModuleMapFile);
+    StringRef externalModuleMapPath;
+    if (externalModuleMapPathRef.takeError()) {
```

Comment:
If you're going to ignore the error, I suggest using `FileManager::getOptionalFileRef()` instead.